Orange/widgets/utils/itemmodels.py

```diff
@@ -869,8 +869,19 @@ class DomainModel(VariableListModel):
                  ATTRIBUTES)
     PRIMITIVE = (DiscreteVariable, ContinuousVariable)
 
-    def __init__(self, order=SEPARATED, placeholder=None,
+    def __init__(self, order=SEPARATED, separators=True, placeholder=None,
                  valid_types=None, alphabetical=False, skip_hidden_vars=True, **kwargs):
+        """
+
+        :param order: How attributes, metas and classes are ordered and where between
```

Comment:
We lately prefer [Napoleon](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html).

Reply:
How attributes (...) -> Order of attributes, metas, classes, separators and other options